pom.xml

```diff
@@ -66,8 +66,7 @@
   </scm>
 
   <properties>
-    <revision>2.30</revision>
-    <changelist>-SNAPSHOT</changelist>
+    <changelist>999999-SNAPSHOT</changelist>
```

Comment:
as this is an API plugin we should not use random versions.  Whilst we have already shot past the version of the bouncycastle plugin we should probably take the time to not make this worse.

You can tie the revision to the bouncycastle version (e.g. remove `bouncycastleVersion` and where it is used in the pom use `revision`  then change `version` to `2.29.`